.github/make-release.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Check for OBS_TOKEN
+if [ -z "${OBS_PROJECT}" ] ; then
+    echo "Enviorment variable OBS_PROJECT must be set"
+    exit 1
+fi
+if [ -z "${OBS_PACKAGE}" ] ; then
+    echo "Enviorment variable OBS_PACKAGE must be set"
+    exit 1
+fi
+if [ -z "${MERLIN_VERSION}" ] ; then
+    echo "Enviorment variable MERLIN_VERSION must be set"
+    exit 1
+fi
+
+# TODO: automate getting platforms from OBS
+PLATFORMS=(CentOS_8_Stream CentOS_7)
+EXPECTED_PACKAGES=${#PLATFORMS[@]}
+
+# Trigger the OBS build
+osc api -X POST "/source/$OBS_PROJECT/$OBS_PACKAGE?cmd=runservice"
+
+echo "Waiting for builds to finish"
+
+# list of possible OBS errors
+errors=(broken unresolvable failed)
+
+# loop until we built all packages or we get an error or max timeout (2hours)
+end=$((SECONDS+7200))
+failed=true
+while [ $SECONDS -lt $end ]
+do
+    sleep 60
+    obs_status=$(osc r $OBS_PROJECT $OBS_PACKAGE)
+    published_count=$(echo $obs_status | grep -o "succeeded" | wc -l)
+    if [ $published_count -ge $EXPECTED_PACKAGES ]
```

Comment:
Are we sure that published_count does not also count old packages that have been built successfully? Looking further down I see that we do some clean up. Just wanting to make sure.

Reply:
There is probably a chance that could happen if either the API call on L22 fails, or if for some reason OBS doesn't change status within 60 seconds. I didn't find any reasonable way to double check this. The best fail safe is probably checking the tag and date of the binaries down the line, but that is also not ideal imo.

I would probably lean towards fixing it when/if it becomes a problem, unless you have a strong opinion otherwise.

Reply:
Agreed. No strong opinion here :smile: